pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -481,7 +481,7 @@ def set_distributed_mode(self):
         # for DDP overwrite nb processes by requested GPUs
         if (
             self._device_type == DeviceType.GPU
-            and self._distrib_type in (DistributedType.DDP, DistributedType.DDP_SPAWN)
+            and self._distrib_type in (DistributedType.DDP, DistributedType.DDP_SPAWN, DistributedType.DDP2)
```

Comment:
@tchaton This does not apply to DDP2, see `tests/trainer/test_trainer.py::test_trainer_config`

Reply:
Yes, I removed it. I messed up there.

Reply:
no, problem, if you remove it, we have all cpu tests passing :) <3

Reply:
I just removed it now